gbpservice/nfp/configurator/drivers/firewall/vyos/vyos_fw_driver.py

```diff
@@ -39,6 +40,73 @@ class FwGenericConfigDriver(object):
     def __init__(self):
         self.timeout = cfg.CONF.rest_timeout
 
+    def _configure_log_forwarding(self, url, mgmt_ip, port):
+        """ Configures Visibility VM IP address in Service VMs to
+            configure log forwarding to the Visibility VM.
+
+            :param url: url format that is used to invoke the Service VM API
+            :param mgmt_ip: management IP of hte Service VM
+            :param port: port that is listened to by the Service VM agent
+
+            Returns: SUCCESS/Error msg
+
+        """
+
+        url = url % (mgmt_ip, port, 'configure-rsyslog-as-client')
+
+        visibility_vm_ip_address = cfg.CONF.visibility_vm_ip_address
+        if not visibility_vm_ip_address:
```

Comment:
We should not fail when visibility VM IP is not configured. This should be treated as non-forwarding case where the deployment don't have visibility VM.